test/users/npadmana/mpi/c2chapel.py

```diff
@@ -0,0 +1,83 @@
+""" A C to Chapel parser to help automate writing bindings.
+
+This is built on top of pycparser :
+  https://github.com/eliben/pycparser
+(pycparser is also part of the Anaconda distribution)
+
+Some parts of this code may need to be special cased for particular bindings,
+and the resulting bindings may require some human intervention.
+"""
```

Comment:
Fixed. The README now has a lot more info. It did require some human intervention, so the description is in English, not a series of commands I ran.